gcloud/storage/blob.py

```diff
@@ -70,9 +72,35 @@ def __init__(self, name, bucket=None):
 
         super(Blob, self).__init__(name=name)
 
+        self._chunk_size = None  # Needs to be defined in __init__.
```

Comment:
Shouldn't need to assign the backing attribute:  the property setter does it just fine.